README.md

```diff
@@ -30,15 +30,14 @@ func main() {
 }
 ```
 
-
 #### Projects using go-prompt
 
-* [c-bata/kube-prompt : An interactive kubernetes client featuring auto-complete written in Go.](https://github.com/c-bata/kube-prompt)
-* [rancher/cli : The Rancher Command Line Interface (CLI)is a unified tool to manage your Rancher server](https://github.com/rancher/cli)
+* [c-bata/kube-prompt : An interactive Kubernetes client featuring auto-complete written in Go.](https://github.com/c-bata/kube-prompt)
+* [rancher/cli : The Rancher command-line interface is a unified tool to manage your Rancher server](https://github.com/rancher/cli)
```

Comment:
These descriptions are copied from the README or description of github repository. So I think we should not correct very much.